### PR TITLE
South Australia battery

### LINKED ISSUE #869: South Australia Battery
Apparently there's data here:
https://ausrealtimefueltype.global-roam.com/expanded

- [x] Add UI
- [x] Add data
- [ ] Add LCA GHG parameters

### PR DESCRIPTION
**Pull request by systemcatch**
The world's largest lithium ion battery is now connected to the South Australian grid!

Ref #869

### CODE REVIEW

**Comment by corradio** [comment]
Wow this is awesome. We'd probably need to make some UI changes to make it appear. I'll work on that asap.

File: parsers/lib/AU_battery.py [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+    """
+
+    #Verify set to false to prevent SSL error due to domain mismatch.
+    req = requests.get(base_url, verify = False)
+    json_content = req.json()
+    dicts = json_content["seriesCollection"]
+    storage_id = (item for item in dicts if item["id"] == "41").next()
+    charging_id = (item for item in dicts if item["id"] == "57").next()
+
+    storage = -1*float(storage_id["value"])
```
Comment (corradio) [inline parsers/lib/AU_battery.py]: Are you sure? storage should be counted positive, and discharge should be counted negative.

Comment (systemcatch) [reply parsers/lib/AU_battery.py]: Storage means it's discharging to the grid, though on reflection that's a confusing variable name.  I'll change it if you want.

**Comment by corradio** [comment]
Let's call it discharge then (and storage when storing, i.e. consuming from
the grid)

**Pull request merged**